Client/MirScenes/GameScene.cs

```diff
@@ -7756,7 +7756,7 @@ public MirControl OverlapInfoLabel(UserItem item, bool Inspect = false)
                     Location = new Point(4, ItemLabel.DisplayRectangle.Bottom),
                     OutLine = true,
                     Parent = ItemLabel,
-                    Text = string.Format(GameLanguage.MaxCombine, realItem.StackSize, GameLanguage.SplitStack)
+                    Text = string.Format(GameLanguage.MaxCombine, realItem.StackSize, "\n")
```

Comment:
remove the \n newline - this should go within the string .MaxCombine so its optional to have a new line / remove it if preferred